GitCommands/AppTitleGenerator.cs

```diff
@@ -46,10 +50,17 @@ public string Generate(string workingDir = null, bool isValidWorkingDir = false,
             var description = _description.Get(workingDir);
 
 #if DEBUG
-            return $"{description} ({branchName}) - Git Extensions [DEBUG]";
+            return $"{description} ({branchName}) - Git Extensions [{_buildBranch}]";
```

Comment:
```suggestion
            _buildBranch = _buildBranch ?? "DEBUG";
            return $"{description} ({branchName}) - Git Extensions [{_buildBranch}]";
```
would be bulletproof.